dotnet/Library/Okapi/ByteBuffer.cs

```diff
@@ -6,7 +6,7 @@ namespace Okapi
     [StructLayout(LayoutKind.Sequential)]
     internal struct ByteBuffer
     {
-        public ulong Length;
+        public long Length;
         public IntPtr Data;
     }
```

Comment:
Is this a .NET-only change or an FFI change?

Reply:
It's a dotnet change that matches the underlying native type, which is `int64`, not `uint64`. Generally it won't effect runtimes even if they're different, except a newer net-6 parser is very strict about this, hence the change.

Reply:
Got it!